roles/logstash/templates/10-filters.conf.j2

```diff
@@ -116,13 +116,24 @@ filter {
             convert => { "apdex" => "float" }
         }
 
+        # Calculate Bandwidth
+        mutate{
+             convert => { "request_length" => "integer" }
+             convert => { "body_sent" => "integer" }
+        }
+        ruby{
+            code =>
+                "bytes_total = event.get('request_length') + event.get('body_sent');
```

Comment:
this is cool, but I think it's nice to see them separate (bytes_sent vs bytes_received)